vespajlib/src/main/java/com/yahoo/tensor/IndexedTensor.java

```diff
@@ -225,6 +225,10 @@ static void indexedBlockToString(IndexedTensor tensor, Indexes indexes, StringBu
                 b.append(tensor.get(index));
             else if (tensor.type().valueType() == TensorType.Value.FLOAT)
                 b.append(tensor.getFloat(index));
+            else if (tensor.type().valueType() == TensorType.Value.BFLOAT16)
+                b.append(tensor.getFloat(index));
```

Comment:
it's probably better to use a switch instead of if-else-if-else here and similar places

Reply:
Thanks, fixed.